modin/backends/pandas/parsers.py

```diff
@@ -284,12 +430,12 @@ def update_row_nums(match):
 
             Parameters
             ----------
-            match
+            match: re.Match object
                 The match from the origin `re.sub` looking for row number tags.
 
             Returns
             -------
-            string
+            str:
```

Comment:
```suggestion
            str
```
I don't think we need a colon after `return type` specification